### PR TITLE
HOLD FOR RELEASE Add support for Kubernetes 1.28

### PR DESCRIPTION
**Pull request by sgalsaleh**

### CODE REVIEW

File: docs/partials/install/_kubernetes-compatibility.mdx [hunk 1]
```diff
@@ -1,5 +1,6 @@
-| KOTS Versions      | Kubernetes Compatibility  |
-|--------------------|---------------------------|
-| v1.100.0 and later  | v1.27, v1.26, v1.25        |
-| v1.93.1 to v1.99.0  | v1.26, v1.25        |
-| v1.87.0 to v1.93.0   | v1.25               |
+| KOTS Versions          | Kubernetes Compatibility    |
+|------------------------|-----------------------------|
+| v1.102.1 and later     | v1.28, v1.27, v1.26, v1.25  |
+| v1.100.0 and v1.102.0  | v1.27, v1.26, v1.25         |
```
Comment (cbodonnell) [inline docs/partials/install/_kubernetes-compatibility.mdx:4]: may need to fix the table
```suggestion
| v1.100.0 to v1.102.0  | v1.27, v1.26, v1.25         |
```

Comment (sgalsaleh) [reply docs/partials/install/_kubernetes-compatibility.mdx]: 🤦‍♂️

**Review by paigecalvert — APPROVED** [review]

**Comment by paigecalvert** [comment]
thanks @sgalsaleh and @cbodonnell !

**Comment by AmberAlston** [comment]
Blocked on ability to release related KOTS 1.28 support

**Pull request merged**